packages/venia-ui/lib/components/Header/cartTrigger.js

```diff
@@ -10,32 +10,33 @@ import CREATE_CART_MUTATION from '../../queries/createCart.graphql';
 import GET_CART_DETAILS_QUERY from '../../queries/getCartDetails.graphql';
 import defaultClasses from './cartTrigger.css';
 
-const CART_ICON_FILLED = (
-    <Icon
-        src={ShoppingCartIcon}
-        attrs={{
-            fill: 'rgb(var(--venia-text))',
-            stroke: 'rgb(var(--venia-text))'
-        }}
-    />
-);
-const CART_ICON_EMPTY = (
-    <Icon
-        src={ShoppingCartIcon}
-        attrs={{
-            stroke: 'rgb(var(--venia-text))'
-        }}
-    />
-);
-
 const CartTrigger = props => {
     const { handleClick, itemCount } = useCartTrigger({
         createCartMutation: CREATE_CART_MUTATION,
         getCartDetailsQuery: GET_CART_DETAILS_QUERY
     });
 
+    const iconColor = props.iconColor;
```

Comment:
Better to destructure here. I can see how you probably just copied the code style from the line below, but that is the one exception to the rule. We like to reserve `classes` for use in the component, so we do not destructure it off props.
```suggestion
    const { iconColor } = props;
```